### PR TITLE
UBERF-7765: Retry config load desktop

### LINKED ISSUE #6215: Desktop app stop working after restart
<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NmFiMmJiNGRhNTY1NDI5ZDY2MjY1NTYiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.DDvqWc5F5lRFKzZo1o5n--KjlSV_r1tWdfjhR3Sacnc">Huly&reg;: <b>UBERF-7765</b></a></sub>

### PR DESCRIPTION
**Pull request by lexiv0re**
* Adds retry when loading config from desktop app
  Related to: https://app.hc.engineering/workbench/platform/tracker/UBERF-7765
  Closes <https://github.com/hcengineering/platform/issues/6215>

Note: I wasn't able to reproduce the issue but hopefully it should fix it.

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NmIyMjUxM2U1NTRjZjllNTY4ZmE3NzIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.lbcUv0Ib_MOt1jb1kNnLw_w_sL_EcEZJuUtRTDWb0iU">Huly&reg;: <b>UBERF-7823</b></a></sub>

### CODE REVIEW

**Review by haiodo — APPROVED** [review]

File: desktop/src/ui/preload.ts [hunk 1]
```diff
@@ -18,6 +17,25 @@ export function concatLink (host: string, path: string): string {
   }
 }
 
+async function loadServerConfig (url: string): Promise<any> {
```
Comment (haiodo) [inline desktop/src/ui/preload.ts:20]: May be we could check for error?

Comment (lexiv0re) [reply desktop/src/ui/preload.ts:20]: You mean only retry for this specific error or somehow handle when all retries fail?

Comment (haiodo) [reply desktop/src/ui/preload.ts:20]: yep

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
@lexiv0re I've merged, so could improve in next PR.

**Pull request merged**